ui/src/js/co2/Co2Calculator.js

```diff
@@ -28,6 +29,7 @@ export default class Co2Calculator extends Component {
       result = 0;
     }
     this.setState({foodResult: result});
+    this.setState({calcButtonActive: true});
   }
```

Comment:
you can collect these changes into one setState method separating the variable with a comma.
something like:
`this.setState({foodResult: result, calcButtonActive: true});`